README.md

```diff
@@ -204,27 +204,27 @@ TiDB (The pronunciation is: /'taɪdiːbi:/ tai-D-B, etymology: titanium) is an o
 
 - __Horizontal scalability__
 
-    TiDB provides horizontal scalability simply by adding new nodes. Never worry about infrastructure capacity ever again.
+    TiDB scales horizontally by simply adding new nodes and there's no sharding required.
```

Comment:
Is there no sharding required, or is sharding automatically managed?  It might be easier on a new user to say it is entirely transparent vs. doesn't happen.